tests/functional/splinter_test.c

```diff
@@ -74,6 +73,17 @@ typedef struct test_splinter_thread_params {
    uint64             seed;
 } test_splinter_thread_params;
 
+/*
+ * Parameters to driver trunk range query performance workload.
+ */
+typedef struct trunk_range_perf_params {
+   const char *range_perf_descr;
+   int         num_ranges;
+   uint64      range_min;
+   uint64      range_max;
+
+} trunk_range_perf_params;
```

Comment:
Struct introduced so we can call range-lookups perf test sub-case with diff min / max / num parameters.